Automattic/jetpack
# Masterbar: Point Sharing menu to Calypso

Fixes #18427.




#### Changes proposed in this Pull Request:

* Replaces Sharing menu item on dotcom with a link to wordpress.com/marketing/sharing-buttons
* Adds unit tests for replaced sharing menu item

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






Jetpack/Atomic sites
* With nav-unification enabled, make sure Settings > Sharing still links to `wp-admin/options-general.php?page=sharing`.

Dotcom
* Apply D55661-code to your sandbox.
* With nav-unification enabled, make sure Settings > Sharing still links to `wordpress.com/marketing/sharing-buttons`.


#### Proposed changelog entry for your changes:


None needed
